master/usage/policy/secure-metrics.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Securing access to Calico's metrics endpoints
+---
+
+## About securing access to {{site.prodname}}'s metrics endpoints
+
+When using {{ site.prodname }} with prometheus metrics enabled, it may be desirable to use network policy
+to limit access to {{ site.prodname }}'s metrics endpoints.
+
+### Purpose of this page
+
+Provide example workflows for creating network policies to limit access to {{site.prodname}}'s metrics endpoints.
+
+## Prerequisites
+
+- {{site.prodname}} is installed with [prometheus metrics reporting enabled](TODO).
+- calicoctl is [installed in your PATH and configured to access the data store](../calicoctl/install).
+
+## Using a blacklist approach
+
+This approach allows all traffic to your hosts by default, but lets you limit access to specific ports using
+Calico policy. This approach allows you to restrict access to access to specific ports, while leaving other
+host traffic unaffected.
+
+### Overview
+
+The basic process is as follows:
+
+1. Create a default network policy which allows traffic to and from your hosts.
+1. Create host endpoints for each node which you'd like to secure.
+1. Create a network policy which denies unwanted traffic to the {{site.prodname}} metrics endpoints.
+1. Apply labels to allow access to the prometheus metrics.
+
+### Example for calico/node
+
+This example shows how to limit access to the {{site.nodecontainer}} prometheus metrics endpoints.
+
+1. Create a default network policy to allow host traffic
+
+   First, create a default-allow policy. It's important to do this first to avoid a drop in connectivity when adding the host endpoints
```

Comment:
Instead of "It's important" I would suggest just saying "You should do this first"

"It's important" is passive and just generally a good phrase to avoid using.